cypress/tests/ui/auth.spec.ts

```diff
@@ -43,12 +44,15 @@ describe("User Sign-up and Login", function () {
     cy.visit("/");
 
     cy.getBySel("signup").click();
+    cy.getBySel("signup-title").should("be.visible").and("contain", "Sign Up");
+    cy.percySnapshot();
 
     cy.getBySel("signup-first-name").type(userInfo.firstName);
     cy.getBySel("signup-last-name").type(userInfo.lastName);
     cy.getBySel("signup-username").type(userInfo.username);
     cy.getBySel("signup-password").type(userInfo.password);
-    cy.getBySel("signup-confirmPassword").type(userInfo.password);;
+    cy.getBySel("signup-confirmPassword").type(userInfo.password);
+    cy.percySnapshot();
```

Comment:
I think you can also give percy snapshots names, like in this case `cy.percySnapshot('about to signup')`

Reply:
Agree and will add.  The initial focus was to get things in place and flowing.